pkg/local_object_storage/shard/gc.go

```diff
@@ -119,24 +98,11 @@ func (gc *gc) listenEvents() {
 			}
 
 			v.prevGroup.Wait()
-
-			v.prevGroup.Add(len(v.handlers))
-
-			for i := range v.handlers {
-				h := v.handlers[i]
-
-				err := gc.workerPool.Submit(func() {
```

Comment:
`drop unused parameters` commit message is not entirely true. We used worker pool, now we don't